repository/m.json

```diff
@@ -355,6 +355,17 @@
 			]
 		},
 		{
+			"name": "Markdown Numbered Headers",
+			"details": "https://github.com/weituotian/md_numbered_headers",
+			"labels": ["markdown", "headers", "numbered headers"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"branch": "master"
```

Comment:
We don't support branch-based releases for new packages. You need to change this line to `"tags": true`.